src/components/Feedback/ErrorPage/ErrorPage.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { faker } from '@faker-js/faker';
+
+import { render, screen, userEvent } from '../../../tests/test-utils';
+import { getErrorContent } from '../../../utils/errors';
+import { ErrorType } from '../../../utils/errors';
+import ErrorPage from '.';
+
+test('Shows default values without props', () => {
+  const defaultError = getErrorContent('Amplify portal', ErrorType.DEFAULT);
+
+  render(
+    <ErrorPage>
+      <ErrorPage.Title />
+      <ErrorPage.Description />
+      <ErrorPage.Action />
+    </ErrorPage>
+  );
+  expect(screen.getByTestId('title').textContent).toEqual(defaultError.title);
```

Comment:
Minor thing but here you can use more specific ".toBe...()" methods like "toHaveTextContent()"